package.json

```diff
@@ -4,9 +4,8 @@
   "description": "Update helmfile version.yaml with latest version tags for each repository",
   "scripts": {
     "lint": "eslint",
-    "test": "jest",
```

Comment:
I sort of disagree with this, I know this is removed because there is no tests, but there should be before this is used in anger. Not a point for your PR, but one for us to either write a ticket or do.